# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/ct/CTVerifyResult.cpp
# Commit: dc509eab3ed7
# Full Hash: dc509eab3ed7a513258723eb46b25f33a610e720
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2025-07-11 16:36:21
# Regressor Bug: 1953221
# File Overlap Count: 3
# Description:
#   Bug 1953221 - don't accept SCTs with a timestamp after the distrustAfter date for the root r=jschanck
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D256720
# ==============================================================================

diff -r 5b86ae2c81f9 -r dc509eab3ed7 security/ct/CTVerifyResult.cpp
--- a/security/ct/CTVerifyResult.cpp	Thu Jul 10 18:37:49 2025 +0000
+++ b/security/ct/CTVerifyResult.cpp	Thu Jul 10 19:04:10 2025 +0000
@@ -27,6 +27,7 @@
   sctsFromUnknownLogs = 0;
   sctsWithInvalidSignatures = 0;
   sctsWithInvalidTimestamps = 0;
+  sctsWithDistrustedTimestamps = 0;
   embeddedSCTs = 0;
   sctsFromTLSHandshake = 0;
   sctsFromOCSP = 0;